backplane-api/Makefile

```diff
@@ -4,12 +4,11 @@
 deploy:
 	@kubectl create namespace aro-hcp --dry-run=client -o json | kubectl apply -f - && \
```

Comment:
This should be ``kubectl create namespace backplane-api --dry-run=client -o json | kubectl apply -f